gems/aws-sigv4/lib/aws-sigv4/signer.rb

```diff
@@ -79,13 +79,15 @@ module SigV4
     #   )
     #
     class Signer
+      # rubocop:disable Style/ClassVars
```

Comment:
Is there a better solution here that rubocop recommends?

Reply:
You can use a class instance variable instead. The behavior is different and implications for inheritance.

In our case, we want a class var and not a class instance variable I believe.